source/features/first-published-tag-for-merged-pr.tsx

```diff
@@ -21,7 +21,8 @@ const getFirstTag = cache.function(async (commit: string): Promise<string | unde
 });
 
 async function init(): Promise<void> {
-	const mergeCommit = select(`.TimelineItem.js-details-container.Details a[href^="/${getRepo()!.nameWithOwner}/commit/" i] > code.link-gray-dark`)!.textContent!;
+	const timelineItemLinkSelector = `.TimelineItem.js-details-container.Details a[href^="/${getRepo()!.nameWithOwner}/commit/" i]`;
+	const mergeCommit = select(`${timelineItemLinkSelector} > code.Link--primary, ${timelineItemLinkSelector} > code.link-gray-dark`)!.textContent!;
```

Comment:
look at #4017 just remove the `.link-gray-dark` I added it when I made the feature for no reason. (Unless you see why its needed)